shell.nix

```diff
@@ -0,0 +1,82 @@
+with import ~/nixpkgs {};
+with pkgs;
```

Comment:
not sure what this file is

Reply:
you can ignore it if you don't run NixOS.